content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/quickstart-for-configuring-your-minio-storage-bucket-for-github-actions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Quickstart for configuring your MinIO storage bucket for GitHub Actions
+intro: 'Configure your custom MinIO storage bucket for use with {% data variables.product.prodname_actions %}.'
+versions:
+  ghes: '*'
+type: quick_start
+topics:
+  - Actions
+  - Enterprise
+  - Infrastructure
+  - Storage
+shortTitle: Quickstart for MinIO
+---
+
+Before you can enable and configure {% data variables.product.prodname_actions %} on {% data variables.product.product_location_enterprise %}, you need to prepare your third-party storage solution.
+
+MinIO offers object storage with support for the S3 API and {% data variables.product.prodname_actions %} on your enterprise.
+
+This quickstart shows you how to set up MinIO using Docker for use with {% data variables.product.prodname_actions %} but you have other options for managing MinIO besides Docker. For more information about MinIO, see the official [MinIO docs](https://docs.min.io/).
+
+## 1. Choose a MinIO mode for your needs
+
+| MinIO mode | Optimized for | Storage infrastructure required |
+|----|----|----|
+| Standalone MinIO (on a single host) | Fast setup |  N/A |
+| Clustered MinIO (also called Distributed MinIO)|  Data security | Storage servers running in a cluster |
+
+For more information about your options, see the official [MinIO docs](https://docs.min.io/).
+
+{% warning %}
+
+**Warning**: MinIO has announced removal of MinIO Gateways. Starting June 1st, 2022, support and bug fixes for the current MinIO NAS Gateway implementation will only be available for paid customers via their LTS support contract. If you want to continue using MinIO Gateways with {% data variables.product.prodname_actions %}, we recommend moving to MinIO LTS support. For more information, see [Scheduled removal of MinIO Gateway for GCS, Azure, HDFS](https://github.com/minio/minio/issues/14331) in the minio/minio repository.
```

Comment:
The name of the deprecated product is MiniO NAS Gateway (not "MinIO Gateways")

Reply:
I'm not sure we need this warning here at all in a new article that has never mentioned MinIO Gateway for NAS. Can we remove it?